### PR TITLE
V0.32.0

### PR DESCRIPTION
**Pull request by carlosalberto**

### CODE REVIEW

**Comment by carlosalberto** [comment]
Hey - created a PR prior to releasing 0.32.0 (including updating the CHANGELOG).

Let me know ;)

**Comment by felixbarny** [comment]
Is this release fully backwards compatible?

**Comment by tylerbenson** [comment]
@felixbarny It should be compatible for users, but implementations will require some changes.

**Comment by carlosalberto** [comment]
Hey @felixbarny yes, it's as @tylerbenson said

As an exception: for users, they will be impacted if they use inject/extract with adapters themselves, i.e.

```java
tracer.inject(context, Format.Builtin.TEXT_MAP, new TextMapInjectAdapter(map));
```

to:

```java
tracer.inject(context, Format.Builtin.TEXT_MAP_INJECT, new TextMapInjectAdapter(map)); // Or...
tracer.inject(context, Format.Builtin.TEXT_MAP, new TextMapAdapter(map));
```

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,19 @@
...
+* Finishing `Span` upon Scope close is deprecated overall.
+* `ScopeManager.active()` is deprecated.
+* `SpanBuilder.startActive()` is deprecated.
+* `AutoFinishScopeManager` is deprecated.
+* Added a new `Binary` format on top of `ByteBuffer`.
+* Added generic typed `setTag()`/`withTag()` versions.
+* Split Inject and Extract builtin interfaces.
+* `Tracer` implements `Closable`.
+* Added `GlobalTracer.registerIfAbsent()`.
+* Added `GlobalTracer.isRegistered()`.
```
Comment (sjoerdtalsma) [inline CHANGELOG.md]: `GlobalTracer.isRegistered()` already existed. The implementation changed.

File: opentracing-api/pom.xml [hunk 2]
```diff
@@ -20,7 +20,7 @@
     <parent>
         <groupId>io.opentracing</groupId>
         <artifactId>parent</artifactId>
-        <version>0.31.1-SNAPSHOT</version>
+        <version>0.32.0-RC3-SNAPSHOT</version>
```
Comment (tylerbenson) [inline opentracing-api/pom.xml]: Should we do an RC3?

Comment (carlosalberto) [reply opentracing-api/pom.xml]: I think we are ready for an actual release - the latest (important) change from RC2 is making `Tracer` implement `Closable` ;)

File: CHANGELOG.md (hunk 1 above)
Comment (carlosalberto) [reply CHANGELOG.md]: Oh, right. Thanks for the review. Will update the Changelog ;)

**Comment by adriancole** [comment]
> Hey @felixbarny yes, it's as @tylerbenson said
> 
> As an exception: for users, they will be impacted if they use inject/extract with adapters themselves, i.e.
> 
> ```java
> tracer.inject(context, Format.Builtin.TEXT_MAP, new TextMapInjectAdapter(map));
> ```
> 
> to:
> 
> ```java
> tracer.inject(context, Format.Builtin.TEXT_MAP_INJECT, new TextMapInjectAdapter(map)); // Or...
> tracer.inject(context, Format.Builtin.TEXT_MAP, new TextMapAdapter(map));
> ```

this should be added to the change log. can you do that? Also, It could be that `TEXT_MAP_EXTRACT` is also new? It is harder to tell without `@since` tags on the apis.

**Comment by tylerbenson** [comment]
@adriancole yes, both `TEXT_MAP_INJECT` and `TEXT_MAP_EXTRACT` were added in this release.

**Pull request merged**